app/templates/events/view/settings/export.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="row">
+ <div class="ui stackable very relaxed container grid">
```

Comment:
for removing space between tab and export page, try to remove "very relaxed container" class

Reply:
Thanks for this suggestion that I wanted to have.